mm2src/mm2_main/src/lp_swap/maker_swap_v2.rs

```diff
@@ -822,6 +824,8 @@ impl<MakerCoin: MmCoin + MakerCoinSwapOpsV2, TakerCoin: MmCoin + TakerCoinSwapOp
             },
         };
 
+        // FIXME: I am lost here, is this the fee for funding spend or taker payment spend?
+        // if it's the former, where is the fee for the the taker payment spend?
         let taker_payment_spend_trade_fee = match state_machine.taker_coin.get_receiver_trade_fee(stage).compat().await
         {
```

Comment:
This is the fee paid by the receiver of a payment, it was an htlc spend fee in the legacy swap protocol. For v2, it should be 0 for maker like we agreed before (but maybe not in this PR, and we make it the taker payment spend fee for now), and for taker it should be the fee needed to spend the maker payment which is the same as the legacy protocol. Is this clear enough?
https://github.com/KomodoPlatform/komodo-defi-framework/blob/3c888aaf513eefe6ffbc99a6f2320dfe566223cb/mm2src/coins/lp_coins.rs#L3102-L3103

Reply:
> and for taker it should be the fee needed to spend the maker payment which is the same as the legacy protocol.

As discussed in DM with @mariocynicys, this should be different than legacy protocol too since the redeem script is a bit different due to the immediate refund path. I will make this in progress again @mariocynicys